service/policy/db/query.sql

```diff
@@ -647,6 +647,26 @@ value_subject_mappings AS (
 	LEFT JOIN subject_condition_set scs ON sm.subject_condition_set_id = scs.id
 	GROUP BY av.id
 ),
+value_resource_mappings AS (
```

Comment:
Will there be a follow-up PR to introduce these new mappings into the attributes protos/DB logic?

Reply:
added tests